### DISCUSSION TITLE
How to install an older version of jaxlib?

### DISCUSSION THREAD

**Discussion by pmchrist** [jax-ml/jax#10691] [Q&A]
For our project we have use an old version of open-spiel which requires jaxlib==0.1.57. The problem is that pip cannot find that version, available ones start with =>0.1.60. How can I install jaxlib==0.1.57? I know that it existed. Thank you in advance!
(project also uses jax==0.2.7, which is available via pip)
![image](https://user-images.githubusercontent.com/36992459/168303589-ded67223-701d-4cd7-8bdf-590b507cefa8.png)

**Comment by YouJiacheng** [comment]
You can find older version in https://storage.googleapis.com/jax-releases/jax_releases.html
Alternatively, `pip install jaxlib==0.1.57 -f https://storage.googleapis.com/jax-releases/jax_releases.html` should work(at least for linux and py3.9).

**Reply by pmchrist** [reply]
It worked, thank you!

**Reply by kartikpodugu** [reply]
I was trying to run dalle-mini model. I am using ubuntu 22.04. I have a GPU, but it has only 12GB VRAM. But, looks like dalle-mini will leverage GPU only when it has more than 24GB VRAM. So, i was trying to run model only CPU and I faced the problem of jaxlib version mismatch. It is resolved using the above command line. Thanks
